src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const field = document.querySelector('.field');
+const countOfRows = field.rows;
```

Comment:
The variable `countOfRows` is intended to store the number of rows, but it is currently storing the rows collection itself. Consider renaming it to `rows` or `fieldRows` to better reflect its purpose.